examples/jsm/exporters/USDZExporter.js

```diff
@@ -488,7 +493,7 @@ function buildMaterial( material, textures ) {
         def Shader "Texture_${ texture.id }_${ mapType }"
         {
             uniform token info:id = "UsdUVTexture"
-            asset inputs:file = @textures/Texture_${ id }.${ isRGBA ? 'png' : 'jpg' }@
+            asset inputs:file = @textures/Texture_${ id }.${ isRGBA ? 'png' : 'jpg' }@ ${mapType === 'normal' ? rawTextureExtra : ''}
```

Comment:
Should something like this be required for metalness, roughness, and other maps?

```suggestion
            asset inputs:file = @textures/Texture_${ id }.${ isRGBA ? 'png' : 'jpg' }@ ${texture.colorSpace === NoColorSpace ? rawTextureExtra : ''}
```

Note the import would need to be added at top of file.

Reply:
I'm not entirely sure but don't think so, I believe it's required for normals because these need to be scaled with 2 * N - 1 (the scale and bias here) and end up outside 0..1 range, so it may not apply to "simply linear" textures. I'll ask!

Reply:
From the docs it looks like it should
https://openusd.org/release/spec_usdpreviewsurface.html#:~:text=sourceColorSpace%20%2D%20token%20%2D%20auto

Reply:
Yeah, I believe it should be what Don is saying. I'll merge and amend it.
We can revert if needed when we hear from the USD Working Group.